authorization/invitation/service/invitation_service.go

```diff
@@ -344,31 +348,32 @@ func (s *invitationServiceImpl) Rescind(ctx context.Context, rescindingUserID, i
 }
 
 // Accept processes an invitation acceptance click, and returns the resource ID of the resource or identity resource which the invitation is for
-func (s *invitationServiceImpl) Accept(ctx context.Context, token uuid.UUID) (string, error) {
-	var resourceID string
+func (s *invitationServiceImpl) Accept(ctx context.Context, token uuid.UUID) (string, string, error) {
+	var resourceID, redirectPath string
 
 	// Locate the invitation
 	inv, err := s.Repositories().InvitationRepository().FindByAcceptCode(ctx, token)
 
 	if err != nil {
-		return resourceID, err
+		return resourceID, redirectPath, err
```

Comment:
can you wrap the error with a more explicit message ? (eg: `errors.Wrapf(err, "could not accept invitation")`)